main.go

```diff
@@ -239,7 +240,7 @@ func RunMark(cCtx *cli.Context) error {
 
 	api := confluence.NewAPI(creds.BaseURL, creds.Username, creds.Password)
 
-	files, err := filepath.Glob(cCtx.String("files"))
+	files, err := doublestar.Glob(os.DirFS("."), cCtx.String("files"))
```

Comment:
Why not use https://github.com/bmatcuk/doublestar#filepathglob ?

Reply:
```suggestion
	files, err := doublestar.FilepathGlob(cCtx.String("files"))
```

Reply:
thanks for the suggestion. I've done that change. FilepathGlob is definitely better